### DISCUSSION TITLE
Another way to use the play button

### DISCUSSION THREAD

**Discussion by kaptainen** [music-assistant/support#1977] [Feature requests and ideas]
![Screenshot_20240304_173137_Chrome](https://github.com/music-assistant/hass-music-assistant/assets/30773649/3b68d73b-50c3-4c40-b59f-e1070ba91afa)

The blue "play" button (in Danish "afspil") 
Have an idea, is it possible that when pressed it would start playing the media without opening the menu with "play now" etc... And that that menu only appears if the button is long pressed or hold..  
If a user listens to alot of different albums or songs they have to press play then play now almost every time they use the button.. I listen to 90% of my music on 1 player and rarely use any of the other items in the menu...  Meaning that I have to confirm with that second press that I want to play the track alot of times during a listening session.. by long press the option is still there but saves me alot of confirms..

**Comment by OzGav** [comment]
We can think about it but the menu that appears also allows selection of the player to use….

**Reply by kaptainen** [reply]
If the menu stays the same as now with the selection of player but menu triggered by a long press.. I would presume that people play more tracks than change player

**Reply by OzGav** [reply]
You can go through your tracks and add to queue so you don’t have to keep doing this  as well of course

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
